### PR TITLE
Integration of P0370 and P0441

### PR DESCRIPTION
**Pull request by CaseyCarter**
...with editorial changes from Issaquah LWG review.

(Skip the rest and just review the latex changes.)

### CODE REVIEW

**Comment by ericniebler** [comment]
Branch "P0459R1-Issaquah-combined-proposal-with-proxy-iterators" has already been merged into master. Please issue this PR against master.

File: ext/omnibus/D0370R3.html [hunk 1]
```diff
@@ -49,12 +49,12 @@
   <tr>
     <td width="172" align="left" valign="top">Document number:</td>
     <td width="435">
-      D0370R2
+      D0370R3
```
Comment (ericniebler) [inline ext/omnibus/D0370R3.html]: Why are you changing the omnibus document? We're done with it.

File: iterators.tex [hunk 2]
```diff
@@ -3148,8 +3148,8 @@
 \begin{itemdescr}
 \pnum
 \removed{\returns}
-\added{\effects Equivalent to }
-\tcode{x.current == y.current}.
+\added{\effects Equivalent to:}
+\tcode{\added{return} x.current == y.current\added{;}}\removed{.}
```
Comment (ericniebler) [inline iterators.tex]: I thought we were producing a clean doc with no change markings.

**Comment by CaseyCarter** [comment]
You've been rewriting history behind my back. I assume everything up to bce0889 is squashed into master?

File: iterators.tex [hunk 3]
```diff
@@ -4036,8 +4036,8 @@
 \begin{itemdescr}
 \pnum
 \removed{\returns}
-\added{\effects Equivalent to }
-\removed{\tcode{std::move}}\added{\tcode{static_cast<reference>}}\tcode{(*current)}.
+\added{\effects Equivalent to:}
+\removed{\tcode{std::move}}\added{\tcode{return static_cast<reference>}}\tcode{(*current);}
```
Comment (ericniebler) [inline iterators.tex]: Shouldn't this be `return iter_move(current);`, or will that be in another PR?

**Comment by CaseyCarter** [comment]
>  I assume everything up to bce0889 is squashed into master?

No, it appears these changes did not make it into the merge.

File: utilities.tex [hunk 4]
```diff
@@ -310,7 +309,7 @@
 \end{itemdecl}
 \begin{itemdescr}
 \pnum
-\effects Equivalent to \tcode{\textit{INVOKE}(std::forward<F>(f), std::forward<Args>(args)...)}~(\cxxref{func.require}).
+\effects Equivalent to: \\ \tcode{return \textit{INVOKE}(std::forward<F>(f), std::forward<Args>(args)...);} (\cxxref{func.require}).
```
Comment (ericniebler) [inline utilities.tex]: Why the `\\`?

File: ext/omnibus/D0370R3.html (hunk 1 above)
Comment (CaseyCarter) [inline ext/omnibus/D0370R3.html]: So that the document in the post-meeting mailing has *exactly* the wording that was incorporated into the working draft, including editorial changes.

**Comment by ericniebler** [comment]
> >I assume everything up to bce0889 is squashed into master?
>
> No, it appears these changes did not make it into the merge.

Oh weird. I have no idea what happened there.

File: utilities.tex (hunk 4 above)
Comment (CaseyCarter) [inline utilities.tex]: I think it reads slightly better with the `return` statement and the cross-reference on the same line.

File: iterators.tex (hunk 3 above)
Comment (CaseyCarter) [inline iterators.tex]: separate PR.

**Comment by CaseyCarter** [comment]
> Oh weird. I have no idea what happened there.

Hence my confusion about the merge to master.

**Comment by ericniebler** [comment]
> No, it appears these changes did not make it into the merge.

They are on master now.
